styles/components/statistic.module.scss

```diff
@@ -0,0 +1,44 @@
+@use "/styles/colors";
+
+.article {
+	background-color: colors.$secondary;
+	width: 17%;
+	height: fit-content;
+	aspect-ratio: 1/1;
+	border-radius: 1vw;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 1%;
+	box-sizing: border-box;
+
+	p {
+		&:nth-child(1) {
```

Comment:
In the case of the circles it make sense to do things this way, as they are the same and change upon their order.
Here you have 2 elements that don't have a common purpose and the difference between does not stem from their order + If one looks at this file, he can't understand for what you differentiate these 2 p elements. 

It would be better differentiate them by either element type (as I commented in a previous comment) or identification (in this case classNames)

Reply:
noted